networkx/algorithms/connectivity/edge_augmentation.py

```diff
@@ -728,7 +728,7 @@ def unconstrained_bridge_augmentation(G):
     implementation implicitly constructs T'.
 
     The leafs of T are the nodes with no existing edges in T'.
-    Order the leafs of T' by DFS prorder. Then break this list in half
+    Order the leafs of T' by DFS preorder. Then break this list in half
```

Comment:
both `preorder` and `pre-order` are used. I generally favor not use `-`s when both forms are acceptable. I didn't make that change in this PR.